apps/prairielearn/src/tests/sync/util.ts

```diff
@@ -15,16 +15,16 @@ interface CourseOptions {
   devModeFeatures: Record<string, boolean> | string[];
 }
 
-interface Tag {
+export interface Tag {
```

Comment:
I'd consider this to be temporary, we'll be able to use a single shared type in #11024.